src/main/java/com/accesso/challengeladder/controller/LoginController.java

```diff
@@ -0,0 +1,34 @@
+package com.accesso.challengeladder.controller;
+
+import com.accesso.challengeladder.requests.LoginRequest;
+import com.accesso.challengeladder.model.User;
+import com.accesso.challengeladder.services.UserService;
+import com.accesso.challengeladder.utils.JsonUtil;
+import com.google.gson.Gson;
+import org.eclipse.jetty.http.HttpStatus;
+import static spark.Spark.post;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+public class LoginController
+{
+    public LoginController() throws SQLException, IOException
+    {
+        UserService userService = new UserService();
+        post("/login", (req, res) -> {
```

Comment:
I wouldn´t worry about changing this one at this point. Endpoints should be always resources, once we have HTTP Basic Authentication in place we could change this to "POST /token", every time we are authenticating a user we´ll be hitting this endpoint to generate a new token.